sechub-pds-core/build.gradle

```diff
@@ -23,6 +23,9 @@ dependencies {
     
     implementation library.springboot_starter_web  // so we got same jackson dependency...
     implementation library.apache_commons_io
+	
+	implementation library.springboot_starter_actuator
```

Comment:
Please have a look if only github displays this incorrectly or if there is a problem with indentation.